transaction_integration_test.go

```diff
@@ -867,3 +867,95 @@ func TestTransactionCreateWithCustomFields(t *testing.T) {
 		t.Fatalf("Returned custom fields doesn't match input, got %q, want %q", x, customFields)
 	}
 }
+
+func TestTransaction3DSCreateTransactionAndSettleSuccess(t *testing.T) {
+	amount := NewDecimal(1007, 2)
+	txn, err := testGateway.Transaction().Create(&TransactionRequest{
+		Type:   "sale",
+		Amount: amount,
+		CreditCard: &CreditCard{
+			Number:         testCreditCards["visa_3ds_success"].Number,
+			ExpirationDate: "01/2020",
+		},
```

Comment:
Thank you for pointing this out. I was working from the Python implementation but the Java one is clearer.